pre_processing/read_slstr_funcs.F90

```diff
@@ -32,15 +32,21 @@
 ! 2020/02/24, SP: Remove the S7/F1 correction, as new processing baseline makes
 !                 this harder to implement, but also makes it less necessary.
 !                 The S7 band now saturates at 312K rather than 305K.
+! 2020/04/14, AP: Add subsetting to the read functions.
 !
 ! Bugs:
 ! SLSTR colocation is poor. Aerosol retrieval unusable. Cloud retrieval suspect.
 !-------------------------------------------------------------------------------
 
+! When defined, the offset between the nadir and oblique views is assumed
+! to be constant. Otherwise, the two longitude fields are read and compared
+! to determine an appropriate offset.
+#define CONSTANT_OBLIQUE_OFFSET 548
```

Comment:
Where has this number come from? Could you add a reference?

Reply:
It comes from my own analysis of SLSTR files as the modal offset found by the other code. I believe it was a Jupyter notebook, that I should be able to print as a PDF. That can't go in the code but it might go in the wiki.

Reply:
Ok, that's probably overkill. Can you just add to the comment saying you foudn this number empirically?